### ISSUE TITLE
[ios] SSL - No CA Chain is set, but required to operate

### ISSUE THREAD

**Issue by kevinresol** [HaxeFoundation/hxcpp#570]
I think this is because _hx_ssl_cert_load_defaults isn't implemented on ios.

**Comment by hughsando** [comment]
I'm not sure where to get the root certificate from on ios.
One option might be to disable the check with Socket.DEFAULT_VERIFY_CERT = false, but this might introduce security issues.

**Comment by kevinresol** [comment]
I think there is no way to read root certs in iOS due to its sandboxed nature.

I think hxcpp can use CFNetwork (available as c++) for SSL connections

**Comment by hughsando** [comment]
Yes, a native solution might be the best way.  hxcpp can inter-op with objc reasonably well now, so a haxe-only solution (no external source code) might be possible.  Pull-requests welcome.

You could also ship your own root certificates, and read them from the assets area.

**Comment by kevinresol** [comment]
> Pull-requests welcome.

I really wished to. But I have totally no idea about c/c++.
Maybe I really have to learn that at some point.

**Comment by alanlanglois** [comment]
Can it be related?
https://github.com/yupswing/akifox-asynchttp/issues/43
